### ISSUE TITLE
`OpenEyeToolkitWrapper` behavior generating conformers with defined or undefined stereochemistry

### ISSUE THREAD

**Issue by mattwthompson** [openforcefield/openff-toolkit#1076]
Some ambiguous behavior can be reached if a molecule is loaded in with `allow_undefined_stereo=True`, which is commonly the case for loading in large molecule datasets and "just let me run MM on this molecule."

* The behavior of `OpenEyeToolkitWrapper` is to give a single stereoisomer is the stereochemistry is defined, but a random (?) stereoisomer if the stereochemistry is not defined. [Here?](https://github.com/openforcefield/openff-toolkit/blob/9d9f93334ded8ca9b2df9d0586d430b3c272c434/openff/toolkit/utils/openeye_wrapper.py#L1689-L1700)
* Not clear what `RKDitToolkitWrapper` does, or even what RDKit itself does.

This can potentially cause other strange behavior, i.e. if OpenEye's failure (after #1050) causes it to be passed to RDKit, which seems to have different behavior.

This points to the value that @lilyminium proposed in https://github.com/openforcefield/openff-toolkit/issues/966, in which the user could have finer control over which wrappers can access which methods.

It may be the case that the only solution here is to document the behavior of the OpenEye wrapper; improving the rest of this behavior is likely a huge task (#495? #1066 #1067) and out of scope for this specific issue. Another open and dormant issue https://github.com/openforcefield/openff-toolkit/issues/669 might be related.

@SimonBoothroyd encountered the OpenEye-specific behavior and I may be recording things incorrectly or erroneous in other ways - feel free to correct me.

cc @j-wags
